sklearn/neighbors/tests/test_kde.py

```diff
@@ -235,3 +236,18 @@ def test_pickling(tmpdir, sample_weight):
     scores_pickled = kde.score_samples(X)
 
     assert_allclose(scores, scores_pickled)
+
+
+def check_estimators_unfitted(name):
```

Comment:
This test is not being run currently because it does not have a `test_` prefix. We can try this:

```py
@pytest.mark.parametrize('method', ['score_samples', 'sample'])
def test_check_is_fitted(method):
    # Check that predict raises an exception in an unfitted estimator.
    # Unfitted estimators should raise a NotFittedError.
    rng = np.random.RandomState(0)
    X = rng.randn(10, 2)
    kde = KernelDensity()

    with pytest.raises(NotFittedError):
        getattr(kde, method)(X)
```